cmd/odf/debug/debug.go

```diff
@@ -0,0 +1,26 @@
+package debug
+
+import (
+	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/spf13/cobra"
+)
+
+// DebugCmd represents the Debug commands
+var DebugCmd = &cobra.Command{
+	Use:                "debug",
```

Comment:
Instead of calling this "debug" mode, how about if we call it "maintenance"? The whole point of taking down the daemon is to do some maintenance operation on it. I believe this will be more easily understood by users than "debug" mode. The debug mode might make sense for developers, but not really for devops. Then all the descriptions can be updated as well to talk about maintenance instead of debugging. I would also vote for this change upstream. But let's discuss more before finalizing.
```suggestion
	Use:                "maintenance",
```

Reply:
sounds good to discuss offline

Reply:
Based on the discussion I have update to `maintenance` but I kept the word `debug` in docs description.

Reply:
IMO the `debug` wording is just confusing for users. I've added some suggestions on new wording, but did not make the changes everywhere, if you could review the rest of the docs for consistency. Also, we may want to make this change upstream first as well and get example output that includes the new wording.